include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -528,6 +512,31 @@ SECTIONS
 	KEEP(*(.gnu.attributes))
 	}
 
+#if defined CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS
+#if CONFIG_ARM_NSC_REGION_BASE_ADDRESS != 0
+	#define NSC_ALIGN . = CONFIG_ARM_NSC_REGION_BASE_ADDRESS
+#elif defined(CONFIG_SOC_SERIES_NRF91X)
```

Comment:
The specific alignment requirements come from the nRF IDAU implementation, so, I would rather change this to the, more generic, CPU_HAS_NRF_IDAU, what do you think?

Reply:
Sure, I have no strong opinion.

Reply:
done